src/django/api/views/auth/user_api_info.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from api.models import (RequestLog, ApiLimit)
+from ...models import Contributor
+
+
+def get_api_call_limit(contributor_id):
```

Comment:
Optinal: move this methods to own custom class/serializer that by initialization will pass **uid**.

Reply:
done